express-notes-api/index.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express');
+const fs = require('fs');
+const data = require('./data.json');
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const notes = [];
+  for (let i = 1; i < data.nextId; i++) {
+    notes.push(data.notes[i]);
+  }
```

Comment:
this will be problematic if there are "gaps" in your notes. try a `for ... in` loop.